l2geth/rollup/sync_service.go

```diff
@@ -595,6 +595,14 @@ func (s *SyncService) applyTransactionToTip(tx *types.Transaction) error {
 	if tx == nil {
 		return errors.New("nil transaction passed to applyTransactionToTip")
 	}
+	// Queue Origin L1 to L2 transactions must have a timestamp that is set by
+	// the L1 block that holds the transaction. This should never happen but is
+	// a sanity check to prevent fraudulent execution.
```

Comment:
Should the be a panic if it should never happen?

Reply:
We cannot panic here - it would be ideal to but we need a db healer after it panics